pkg/repositories/transformers/workflow.go

```diff
@@ -47,11 +48,26 @@ func FromWorkflowModel(workflowModel models.Workflow) (admin.Workflow, error) {
 		return admin.Workflow{}, errors.NewFlyteAdminErrorf(codes.Internal, "failed to read created at timestamp")
 	}
 
+	var workflowInterface core.TypedInterface
+	if workflowModel.TypedInterface != nil && len(workflowModel.TypedInterface) > 0 {
```

Comment:
no need to check nil here, len should be sufficient! https://pkg.go.dev/builtin#len